docs/core/diagnostics/dotnet-stack.md

```diff
@@ -0,0 +1,121 @@
+---
+title: dotnet-stack tool - .NET Core
+description: Installing and using the dotnet-stack command-line tool.
+ms.date: 11/17/2020
+---
+# Inspect managed stack traces (dotnet-stack)
+
+**This article applies to:** ✔️ .NET Core 3.0 SDK and later versions
+
+## Install
+
+There are two ways to download and install `dotnet-stack`:
+
+- **dotnet global tool:**
+
+  To install the latest release version of the `dotnet-counters` [NuGet package](https://www.nuget.org/packages/dotnet-stack), use the [dotnet tool install](../tools/dotnet-tool-install.md) command:
+
+  ```dotnetcli
+  dotnet tool install --global dotnet-stack
+  ```
+
+- **Direct download:**
+
+  Download the tool executable that matches your platform:
+
+  | OS  | Platform |
+  | --- | -------- |
+  | Windows | [x86](https://aka.ms/dotnet-stack/win-x86) \| [x64](https://aka.ms/dotnet-stack/win-x64) \| [arm](https://aka.ms/dotnet-stack/win-arm) \| [arm-x64](https://aka.ms/dotnet-stack/win-arm64) |
+  | macOS   | [x64](https://aka.ms/dotnet-stack/osx-x64) |
+  | Linux   | [x64](https://aka.ms/dotnet-stack/linux-x64) \| [arm](https://aka.ms/dotnet-stack/linux-arm) \| [arm64](https://aka.ms/dotnet-stack/linux-arm64) \| [musl-x64](https://aka.ms/dotnet-stack/linux-musl-x64) \| [musl-arm64](https://aka.ms/dotnet-stack/linux-musl-arm64) |
+
+## Synopsis
+
+```console
+dotnet-stack [-h, --help] [--version] <command>
+```
+
+## Description
+
+The `dotnet-stack` tool:
+
+* Is a cross-platform .NET Core tool.
+* Captures and prints the managed stacks for all threads in the target .NET process.
+* Is built on [`EventPipe`](./eventpipe.md) of the .NET Core runtime.
+
+## Options
+
+- **`-h|--help`**
+
+  Shows command-line help.
+
+- **`--version`**
+
+  Displays the version of the dotnet-stack utility.
+
+## Commands
+
+| Command                                                   |
+|-----------------------------------------------------------|
+| [dotnet-stack report](#dotnet-stack-report)               |
+| [dotnet-stack ps](#dotnet-stack-ps)                       |
+
+## dotnet-stack report
+
+Collects a diagnostic trace from a running process.
+
+### Synopsis
+
+```console
+dotnet-stack report -p|--process-id <pid>
+                    -n|--name <process-name>
```

Comment:
This is just an aside and you don't need to change anything. I know dotnet-stack didn't start the trend but I find the name of the "--name" option is super ambiguous : ) Without docs I would never guess what this meant.